pkg/helm/controller.go

```diff
@@ -30,7 +30,7 @@ import (
 var (
 	trueVal         = true
 	commaRE         = regexp.MustCompile(`\\*,`)
-	DefaultJobImage = "rancher/klipper-helm:v0.5.0-build20210505"
+	DefaultJobImage = "rancher/klipper-helm:v0.6.0-build20210608"
```

Comment:
Can this be a const?

Reply:
No, we just made it a var in https://github.com/k3s-io/helm-controller/pull/96